src/murmur/Meta.cpp

```diff
@@ -428,8 +428,8 @@ void MetaParams::read(QString fname) {
 			qFatal("Failed to find certificate matching private key.");
 		}
 		if (ql.size() > 0) {
-			QSslSocket::addDefaultCaCertificates(ql);
-			qCritical("Adding %d CA certificates from certificate file.", ql.size());
+			qlIntermediates = ql;
+			qCritical("Adding %d intermediate certificates from certificate file.", ql.size());
```

Comment:
> Is the root CA cert also considered "intermediate"?
> Isn't the root cert public key also part of the key chain in the PEM, so "intermediate" is only true for the others still in the list?

No, typically you do't put the root certificate into the PEM bundle, since it's supposed to be on clients's systems.

You can do it, but it is not necessary. It doesn't hurt us if users do it.

> (We could remove the root cert (which we also have in qlCA) from the list above where we also remove our leaf cert.)

As mentioned, you aren't supposed to put the root cert in your bundle.
Also, we don't know what the root cert *is*. That would require us to do stricter checks than we do now.
And no, it is not required to have the root cert in the qlCA list in MetaParams. (There is no other qlCA in the source tree of this PR -- the 'other' one is now qlIntermediates.)